client/components/domains/register-domain-step/index.jsx

```diff
@@ -369,8 +369,13 @@ class RegisterDomainStep extends React.Component {
 		} );
 
 		const timestamp = Date.now();
-		const testGroup = abtest( 'domainSuggestionTestV5' );
-		if ( 'group_1' === testGroup || 'group_2' === testGroup || 'group_3' === testGroup ) {
+		const testGroup = abtest( 'domainSuggestionTestV6' );
+		if (
```

Comment:
It would be better to do `includes( ['group_1', 'group_2', 'group_3' ,'group_4'], testGroup)` here - that'll check if the test variation is one of the array values